### ISSUE TITLE
[Event Request] Codeunit 23 Item Jnl.-Post Batch - OnPostLinesOnBeforeWindowUpdate

### ISSUE THREAD

**Issue by VMotin** [microsoft/ALAppExtensions#11462]
Hi, would it be possible to get an event OnPostLinesOnBeforeWindowUpdate in codeunit 23, procedure PostLines, like this : 

```
    local procedure PostLines(var ItemJnlLine: Record "Item Journal Line"; var PhysInvtCountMgt: Codeunit "Phys. Invt. Count.-Management")
    var
        TempTrackingSpecification: Record "Tracking Specification" temporary;
        OriginalQuantity: Decimal;
        OriginalQuantityBase: Decimal;
    begin
        LastDocNo := '';
        LastDocNo2 := '';
        LastPostedDocNo := '';
        with ItemJnlLine do begin
            SetCurrentKey("Journal Template Name", "Journal Batch Name", "Line No.");
            FindSet;
            repeat
                if not EmptyLine and
                   (ItemJnlBatch."No. Series" <> '') and
                   ("Document No." <> LastDocNo2)
                then
                    TestField("Document No.", NoSeriesMgt.GetNextNo(ItemJnlBatch."No. Series", "Posting Date", false));
                if not EmptyLine then
                    LastDocNo2 := "Document No.";
                MakeRecurringTexts(ItemJnlLine);
                ConstructPostingNumber(ItemJnlLine);

                OnPostLinesOnBeforePostLine(ItemJnlLine, SuppressCommit);

                if "Inventory Value Per" <> "Inventory Value Per"::" " then
                    ItemJnlPostSumLine(ItemJnlLine)
                else
                    if (("Value Entry Type" = "Value Entry Type"::"Direct Cost") and ("Item Charge No." = '')) or
                       (("Invoiced Quantity" <> 0) and (Amount <> 0))
                    then begin
                        LineCount := LineCount + 1;

                        IsHandled := false;
                        OnPostLinesOnBeforeWindowUpdate(IsHandled);
                        if not IsHandled then begin
                            Window.Update(3, LineCount);
                            Window.Update(4, Round(LineCount / NoOfRecords * 10000, 1));
                        end;
                        

```

```
    [IntegrationEvent(false, false)]
    local procedure OnPostLinesOnBeforeWindowUpdate( var IsHandled: Boolean)
    begin
    end;

```
?
Thanks

**Comment by AlexanderYakunin** [comment]
Why do you need this?

**Comment by Duffy77** [comment]
I think this Problem was fixed with Version 17.4
![image](https://user-images.githubusercontent.com/76178557/126206470-efa44db4-e729-4b48-805a-a8444237bbc7.png)
